cmd/tokenrefresh/main.go

```diff
@@ -0,0 +1,352 @@
+// Copyright 2024 Undistro Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"os"
+	"os/signal"
+	"reflect"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/undistro/zora/pkg/authentication"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	secretName       string
+	secretNamespace  string
+	tokenName        string
+	domain           string
+	clientID         string
+	minRefreshTime   time.Duration
+	refreshThreshold time.Duration
+	kubeconfig       string
+)
+
+const (
+	annotation_status = "status"
```

Comment:
Prefer camel case instead of snake case for Go variables.

We can also prefix this annotation, similar to `zora.undistro.io/inject-conversion` we already have.
What do you think?

```suggestion
	annotationStatus = "zora.undistro.io/status"
```

Reply:
I've pushed up the change